tests/integration/targets/docker_swarm_service/tasks/tests/options.yml

```
@@ -1313,6 +1345,19 @@
       - mode_2 is not changed
       - mode_3 is changed
 
+- assert:
+    that:
+      - mode_4 is changed
+      - mode_5 is not changed
+      - mode_6 is changed
```

Failed tasks are also not changed, so if testing for 'not changed' better also test for 'not failed':
```suggestion
      - mode_4 is changed
      - mode_5 is not changed and mode_5 is not failed
      - mode_6 is changed
```